memo_app.rb

```diff
@@ -0,0 +1,117 @@
+# frozen_string_literal: true
+
+require 'sinatra'
+require 'sinatra/reloader'
+require 'json'
+require 'securerandom'
+require 'rack/session/cookie'
+
+configure do
+  enable :sessions
+  use Rack::Session::Cookie, key: 'rack.session', path: '/', secret: 'your_secret_key'
+end
+
+helpers do
+  def memo_data_json_file_path(id)
+    "json/memos_#{id}.json"
+  end
+
+  def h(text)
+    Rack::Utils.escape_html(text)
+  end
+
+  def make_id
+    SecureRandom.uuid
+  end
+
+  def get_memo(id)
+    memo_file_path = memo_data_json_file_path(id)
+    return unless File.exist?(memo_file_path)
```

Comment:
`nil` を返すようにしているのなら、そうであることが分かる明示的に書きましょう。

```suggestion
    return nil unless File.exist?(memo_file_path)
```

Reply:
nilがわかるように明示的に記載いたしました